src/components/Home/styles.js

```diff
@@ -0,0 +1,90 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+export default makeStyles((theme) => ({
+    score: {
+        width: '100vw',
+        height: '52.57vw',
+        backgroundImage: 'url(/images/homeBackground.png)',
+        backgroundSize: 'contain',
+        marginTop:'-1%'
+    },
+    heading: {
+        fontSize: '8vw',
+        fontFamily: 'Source Sans Pro',
+        fontStyle: 'normal',
+        fontWeight: '900',
+        color: '#FFC220',
```

Comment:
use colors by importing from `styles/colors.js` 
Add if any color doesn't exists in that file